x/oracle/abci.go

```diff
@@ -38,16 +38,14 @@ func CalcPrices(ctx sdk.Context, params types.Params, k keeper.Keeper) error {
 	validatorClaimMap := make(map[string]types.Claim)
 	powerReduction := k.StakingKeeper.PowerReduction(ctx)
 	// Calculate total validator power
-	var totalBondedValidatorPower int64
+	var totalBondedPower int64
 	for _, v := range k.StakingKeeper.GetBondedValidatorsByPower(ctx) {
```

Comment:
These two consecutive loops of `for _, v := range k.StakingKeeper.GetBondedValidatorsByPower(ctx)` can be combined

Reply:
pushed a commit which combines them

Reply:
Yes, another old artifact to improve on.